forge-gui/res/cardsfolder/upcoming/protean_war_engine.txt

```diff
@@ -0,0 +1,12 @@
+Name:Protean War Engine
+ManaCost:R W
+Types:Artifact Vehicle
+PT:0/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraft | TriggerDescription$ When CARDNAME enters the battlefield, draft a card from CARDNAME's spellbook and exile it.
```

Comment:
This should be a replacement effect... see other cards with exact wording "As CARDNAME enters the battlefield,"

Though it's a little tough to say because this TriggerDescription doesn't match Oracle

Reply:
You're right. Will fix when I'm home
![image](https://user-images.githubusercontent.com/67333662/194115467-1911715f-2273-473d-8f80-3209e865e17a.png)